iam/deny/update_deny_policy.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START iam_update_deny_policy]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	iam "cloud.google.com/go/iam/apiv2"
+
+	iampb "google.golang.org/genproto/googleapis/iam/v2"
+	"google.golang.org/genproto/googleapis/type/expr"
+)
+
+// updateDenyPolicy updates the deny rules and/ or its display name after policy creation.
+func updateDenyPolicy(w io.Writer, projectID, policyID, etag string) error {
```

Comment:
Should this sample explain (or demonstrate, by showing `GetPolicy()`) how to obtain the `etag`?

Reply:
I guess etag it's not main point here, we just show how to actually call GetPolicy
python samples for comparision: https://github.com/googleapis/python-iam/blob/main/samples/snippets/get_deny_policy.py#L49